src/parquet/util/bit-stream-utils.inline.h

```diff
@@ -86,34 +89,106 @@ inline bool BitWriter::PutVlqInt(uint32_t v) {
 }
 
 template <typename T>
+inline void GetValue_(
+  int num_bits, T* v, const uint8_t* buffer, int& bit_offset,
+  int& byte_offset, uint64_t& buffered_values, int max_bytes) {
```

Comment:
The indentation here is a bit off, can you run `make format` if you have LLVM / clang-tools installed?